packages/binding-mqtt/src/mqtt-broker-server.ts

```diff
@@ -246,42 +246,22 @@ export default class MqttBrokerServer implements ProtocolServer {
          * https://github.com/mqttjs/MQTT.js/pull/1103
          * For further discussion see https://github.com/eclipse-thingweb/node-wot/pull/253
          */
-        let value;
-
+        let contentType = ContentSerdes.DEFAULT;
```

Comment:
I think this and the following lines could be simplified like so:

```suggestion
        const contentType = packet?.properties?.contentType ?? ContentSerdes.DEFAULT;
```